### ISSUE TITLE
Fix str::from_utf8(...).unwrap()

### ISSUE THREAD

**Issue by stanislav-tkach** [stanislav-tkach/os_info#300]
See https://github.com/stanislav-tkach/os_info/pull/295 for details.

**Comment by martintc** [comment]
Oh okay, was under the assumption that if an error occured, it would hit `_ =>`. So, since `from_utf8` returns a result, can we cast aside the `unwrap()` and instead do something like:

```
 match str::from_utf8(&os.stdout) {
        Ok("FreeBSD\n") => { // some code},
        Err(_) => Type::FreeBSD
}
```

**Comment by martintc** [comment]
Ill get the implemented in code. I also noticed I can probably call `trim()` and remove the new line characters in the string literals, making the code look a little cleaner.

**Comment by stanislav-tkach** [comment]
Closed with #303.
